public/css/main.css

```diff
@@ -23,16 +23,16 @@
   -moz-osx-font-smoothing: grayscale;
 }
 
-.dcricon-qrcode:before {
+.exccicon-qrcode:before {
   content: "\e9f8";
 }
-.dcricon-sun-fill:before {
+.exccicon-sun-fill:before {
   content: "\e901";
 }
-.dcricon-sun-stroke:before {
+.exccicon-sun-stroke:before {
   content: "\e902";
 }
-.dcricon-decred:before {
+.exccicon-excc:before {
```

Comment:
Consider removing this class since its related to Decred logo and should not be used